website/docs/tutorials/06-deploying/02-migrations/README.md

```diff
@@ -5,30 +5,30 @@ sidebar_label: Migrations
 # Migrations
 
 :::warning
-This feature is actively developed. Many new additions or changes will land in future Protostar releases.
+Breaking changes can be introduced without depreciation. StarkNet [deployment flow can change](https://community.starknet.io/t/universal-deployer-contract-proposal/1864) so do Protostar.
 :::
 
 
-Migrations are Cairo files that help you manage contracts on the StarkNet. They are especially useful, when your project consists of multiple contracts. Migration code is mainly written in hints. Protostar injects special functions into hints' scope, which you can use to interact with the StarkNet. These functions are similar to cheatcodes used to test contracts, hence in this document we will refer to these functions as migration cheatcodes.
-
-:::info
-Protostar aims to make migration cheatcodes a subset of [testing cheatcodes](/docs/tutorials/testing/cheatcodes) in order to allow testing migration scripts against Protostar's local StarkNet.
-:::
+Migrations are Cairo files that help you manage contracts on the StarkNet.
+These files are responsible for staging your deployment tasks, and they're written under the assumption that your project uses [Proxy Pattern](https://blog.openzeppelin.com/proxy-patterns/).
+As your project evolves, you'll create new migration scripts to further this evolution on the blockchain.
```

Comment:
I don't like this phrasing. I wouldn't rely on anyone understanding usage of further as a verb

Reply:
[update the blockchain sentence](https://github.com/software-mansion/protostar/pull/927/commits/92a45b5930e2659071850f5180add853b3f39a69)